.jenkins/scripts/global/wait-dpkg.sh

```diff
@@ -0,0 +1 @@
+while sudo lsof /var/lib/dpkg/lock-frontend | grep dpkg; do sleep 3; done
```

Comment:
We should change this to 
 our more thorough script, but since we're out of time for this migration we should leave it to later. https://github.com/openenclave/openenclave/blob/master/.jenkins/library/vars/helpers.groovy#L13 or https://github.com/openenclave/openenclave/blob/master/scripts/ansible/install-ansible.sh#L10